Blahgger/Blahgger/Global.asax.cs

```diff
@@ -13,6 +15,39 @@ protected void Application_Start()
         {
             AreaRegistration.RegisterAllAreas();
             RouteConfig.RegisterRoutes(RouteTable.Routes);
+        }
+
+        protected void RegisterGlobalFilters(GlobalFilterCollection filters)
+        {
+            filters.Add(new HandleErrorAttribute());
+            filters.Add(new System.Web.Mvc.AuthorizeAttribute());
+        }
```

Comment:
You're almost there with adding your global filters... you just need to call the `RegisterGlobalFilters` method in the `Application_Start` method.